game.js

```diff
@@ -1238,6 +1238,11 @@ class GameManager extends EventTarget {
           .add(localVector2.set(0, 0.5, -1).applyQuaternion(localPlayer.quaternion)),
         dropDirection: zeroVector, */
       });
+      app.dispatchEvent({
```

Comment:
This looks almost correct; we should only "unuse" the app if it was being used. That code can be abstracted instead of trying to manage use event emits in the `dropSelectedApp` method.

Reply:
Alright, I will use _endUse() instread, seems that function was already validating if the app was being used.